utils/model_uploader/model_autotracing.py

```diff
@@ -0,0 +1,452 @@
+    :type pooling_mode: string
+    """
+    folder_path = (
+        TORCHSCRIPT_FOLDER_PATH
+        if model_format == TORCH_SCRIPT_FORMAT
+        else ONNX_FOLDER_PATH
+    )
+
+    # 1.) Initiate a sentence transformer model class object
+    pre_trained_model = None
+    try:
+        pre_trained_model = SentenceTransformerModel(
+            model_id=model_id, folder_path=folder_path, overwrite=True
+        )
+    except Exception as e:
+        assert (
+            False
+        ), f"Raised Exception in tracing {model_format} model\
+                             during initiating a sentence transformer model class object: {e}"
+
+    # 2.) Save the model in the specified format
+    model_path = None
+    try:
+        if model_format == TORCH_SCRIPT_FORMAT:
+            model_path = pre_trained_model.save_as_pt(
+                model_id=model_id, sentences=TEST_SENTENCES
+            )
+        else:
+            model_path = pre_trained_model.save_as_onnx(model_id=model_id)
+    except Exception as e:
+        assert False, f"Raised Exception during saving model as {model_format}: {e}"
+
+    # 3.) Create a model config json file
+    try:
+        pre_trained_model.make_model_config_json(
+            version_number=model_version,
+            model_format=model_format,
+            embedding_dimension=embedding_dimension,
+            pooling_mode=pooling_mode,
+        )
+    except Exception as e:
+        assert (
+            False
+        ), f"Raised Exception during making model config file for {model_format} model: {e}"
+
+    # 4.) Return model_path & model_config_path for model registration
+    model_config_path = folder_path + MODEL_CONFIG_FILE_NAME
+    return model_path, model_config_path
+
+
+def register_and_deploy_sentence_transformer_model(
+    ml_client: "MLCommonClient",
+    model_path: str,
+    model_config_path: str,
+    model_format: str,
+) -> List["DTypeLike"]:
+    """
+    Register the pretrained sentence transformer model by using the model file and the model config file,
+    deploy the model to generate embeddings for the TEST_SENTENCES,
+    and return the embeddings for model verification
+
+    :param ml_client: A client that communicates to the ml-common plugin for OpenSearch
+    :type ml_client: MLCommonClient
+    :param model_path: Path to model file
+    :type model_path: string
+    :param model_config_path: Path to model config file
+    :type model_config_path: string
+    :param model_format: Model format ("TORCH_SCRIPT" or "ONNX")
+    :type model_format: string
+    """
+    embedding_data = None
+
+    # 1.) Register & Deploy the model
+    model_id = ""
+    try:
+        model_id = ml_client.register_model(
+            model_path=model_path,
+            model_config_path=model_config_path,
+            deploy_model=True,
+            isVerbose=True,
+        )
+        print()
+        print(f"{model_format}_model_id:", model_id)
```

Comment:
Can't we add \n here and then remove the `print()`?

Reply:
Sure